tests/aws/test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 The Go Cloud Authors
```

Comment:
As I mentioned offline, I don't think the test scripts should be converted, since it's mostly maintainers.  I agree with the samples being generally runnable on different platforms, but this is going beyond original scope. Please revert.

Reply:
Done.